files/chef-marketplace-cookbooks/chef-marketplace/templates/default/reckoner.rb.erb

```diff
@@ -1,16 +1,17 @@
 updater.driver = '<%= node['chef-marketplace']['platform'] == 'aws' ? 'ec2' : node['chef-marketplace']['platform'] %>'
 checker.driver = '<%= node['chef-marketplace']['role'] == 'aio' ? 'chef_server' : 'compliance' %>'
-license.count = <%= node['chef-marketplace']['license_count'] %>
+license.count = <%= count = node['chef-marketplace']['license_count']; count.is_a?(String) ? "'#{count}'" : count %>
```

Comment:
Curious as to when it would be a String (and what would the value be?) vs just forcing this to be an integer.